mfr_pdb/render.py

```diff
@@ -1,11 +1,12 @@
 """Molecule renderer module """
-
+#Uses PV: https://github.com/biasmv/pv
 import os
 import mfr
 from mfr.core import RenderResult, get_assets_from_list
 
 # assets must be loaded in this order
 JS_ASSETS = [
+    #The OSF has jquery on each page, uncomment if necessary (e.g. for the player)
```

Comment:
The user should not have to modify any source files manually. There should either be a config option to include jquery, or add a note in the documentation that jquery is required for this renderer.